.github/workflows/benchmarks.yml

```
@@ -101,9 +110,23 @@ jobs:
           terraform_version: 1.3.7
           terraform_wrapper: false
 
+      - name: Init terraform module
+        run: |
+          AWS_REGION=${{ env.AWS_REGION }}
+          echo "TF_VAR_worker_region=$AWS_REGION" >> "$GITHUB_ENV"
+          echo "TF_VAR_run_standalone=$RUN_STANDALONE" >> "$GITHUB_ENV"
+          make init
+
       - name: Build apmbench
         run: make apmbench $SSH_KEY terraform.tfvars
 
+      - name: Build APM Server and Moxy
+        run: |
+          if [ "$RUN_STANDALONE" = true ]; then
+            make apm-server
+            make moxy
+          fi
```

The condition might help to know if the step has been triggered or no easily:

```suggestion
        if: env.RUN_STANDALONE == 'true'
        run: |
          make apm-server
          make moxy
```

---

Unfortunately, Github Actions do no support condition on env variables natively. There is a workaround to pass it as a step output, let me try to check if it works better.

---

Updated via steps outputs now.